travvy/src/components/Home.vue

```diff
@@ -89,12 +89,13 @@
      }
    },
    
-
   methods: {
+    // This sends a request, when the search button, is clicked to recommend to retrieve recommended attractions based on location and reroutes to the attractions list page 
```

Comment:
I think that this was a great place to add logic explanation, since the method is referenced in the code above but might not be super clear